contracts/TokenSale.sol

```diff
@@ -131,18 +162,38 @@ contract TokenSale is Ownable {
             _buyers.length == _newTokenOutAmounts.length,
             "TokenSale: parameter length mismatch"
         );
-        require(block.timestamp < saleStart, "TokenSale: sale already started");
+        require(block.timestamp < saleStart, "TokenSale: ongoing sale");
```

Comment:
true, merely the order of functions called. since saleRecipient is a multisig, i think having the flexibility on the order of functions called might be beneficial